install/roles/network/vars/main.yml

```diff
@@ -0,0 +1,3 @@
+---
+
+network_name: pbench_network
```

Comment:
Why do we need the network named with "pbench" in it?

Reply:
network_name default value is pbench_network but the user can update any name of his choice

Reply:
/home/jaison/scale/jetpack/install/boot_vms.yml:90 tries this by default:
            "_raw_params": ". /home/stack/overcloudrc\necho 'centos' | sshpass ssh-copy-id -i /home/stack/.ssh/id_rsa -o 'StrictHostKeyChecking no' -f root@pbench_network=10.0.0.253\n", 
Not sure why its taken this. I have set trafficgen host ip as:
  ip: 192.168.24.27